huaweicloud/services/iec/resource_huaweicloud_iec_vpc_subnet.go

```diff
@@ -0,0 +1,258 @@
+package iec
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/iec/v1/subnets"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func resourceIecSubnetDNSList(d *schema.ResourceData) []string {
```

Comment:
use buildxxx will be more clear

```suggestion
func buildIecSubnetDNSList(d *schema.ResourceData) []string {
```